ui.apps/src/main/content/jcr_root/apps/core/fd/components/form/wizard/v1/wizard/wizard.html

```diff
@@ -60,15 +60,17 @@
                 aria-label="Previous Button">
         </div>
 
-        <div
-            data-sly-repeat.child="${resource.listChildren}"
-            role="tabpanel"
-            class="cmp-adaptiveform-wizard__wizardpanel"
-            aria-labelledby="${child.name}_wizard-item-nav"
-            data-cmp-hook-adaptiveFormWizard="wizardpanel">
-            <sly data-sly-resource="${child.name}">
-            </sly>
-        </div>
+        <sly data-sly-repeat.child="${resource.listChildren}">
+            <div
+                data-sly-test="${!('fd:' in child.name)}"
```

Comment:
we should check startswith here. if its not OOTB function in sightly then we should create one func.

Reply:
Hi @nit23uec , Implemented starts with check with including the logic in a js util file and placing it in commons. Used the same file in htl by use api.
I didn't implement by using @ resource type in data-sly-resource as it was causing different issues for rendering because of difference in node path and the "name" parameter of out resource.